tests/lib/devicetree/api/src/main.c

```diff
@@ -750,6 +750,17 @@ ZTEST(devicetree_api, test_irq)
 	zassert_true(DT_INST_IRQ_HAS_NAME(0, stat), "");
 	zassert_true(DT_INST_IRQ_HAS_NAME(0, done), "");
 	zassert_false(DT_INST_IRQ_HAS_NAME(0, alpha), "");
+
+#ifdef CONFIG_MULTI_LEVEL_INTERRUPTS
```

Comment:
we should also make sure that the macro works as expected on such node for non-`CONFIG_MULTI_LEVEL_INTERRUPTS` build

Reply:
fixed